notebooks/Basic example.ipynb

```diff
@@ -23,7 +41,15 @@
     "    path = os.path.abspath(path)\n",
     "    with open(path, \"r\") as f:\n",
     "        pg = json.load(f)\n",
-    "    return pg\n"
+    "    return pg"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "607eb405",
+   "metadata": {},
+   "source": [
+    "Load process graph, convert to evalscript and print generated evalscript."
```

Comment:
We could add the `jsbeautifier` here too, to fix up the evalscript formatting a bit.

Reply:
Agree, added